docs/farming.md

```diff
@@ -35,11 +35,11 @@ Install [Polkadot.js extension](https://polkadot.js.org/extension/) into your br
 The address of your account will be necessary at the last step.
 
 ## Required ports
-Currently, TCP port `30333` needs to be exposed for node to work properly.
+Currently, TCP ports `30333` and `40333` need to be exposed for node and farmer to work properly.
 
-If you have a server with no firewall, there is nothing to be done, but otherwise make sure to open TCP port `30333` for incoming connections.
+If you have a server with no firewall, there is nothing to be done, but otherwise make sure to open TCP ports `30333` and `40333` for incoming connections.
 
-On the desktop side if you have a router in front of your computer, you'll need to forward TCP port `30333` to the machine on which your node is running (how this is done varied from router to router, but there is always a feature like this, ask [on the forum](https://forum.subspace.network/) if you have questions).
+On the desktop side if you have a router in front of your computer, you'll need to forward TCP ports `30333` and `40333` to the machine on which your node is running (how this is done varied from router to router, but there is always a feature like this, ask [on the forum](https://forum.subspace.network/) if you have questions).
```

Comment:
I believe 40333 is for farmer network. We don't have that currently in Desktop. 
But will have. 
Just wanted to note that :)

Reply:
It isn't really used by default in CLI either, but it will so would be better to prepare people and have port open already (the port is listening by default, just will not be actually used for queries yet)